packages/appsync-modelgen-plugin/src/utils/process-belongs-to.ts

```diff
@@ -55,9 +60,10 @@ export function processBelongsToConnection(
   return {
     kind: CodeGenConnectionType.BELONGS_TO,
     connectedModel: otherSide,
-    isConnectingFieldAutoCreated,
+    isConnectingFieldAutoCreated: false,
     targetName: targetNames[0],
     targetNames,
+    reference,
```

Comment:
naming nit for the variable and the `CodeGenFieldConnection` type: `useReferences` or `isUsingReferences`?